src/main/java/mServer/crawler/sender/MediathekArd.java

```diff
@@ -236,82 +252,101 @@ private void filmSuchen1(String strUrlFeed, String thema, boolean weiter) {
+                    {
                         break;
                     }
                 }
                 pos += MUSTER_FILM_SUCHEN1.length();
                 url = seite1.extract("documentId=", "&", pos);
-                if (url.contains("\"")) {
-                    url = url.substring(0, url.indexOf('\"'));
+                if (url.contains("\""))
+                {
+                    url = url.substring(0, url.indexOf("\""));
                 }
-                if (!url.isEmpty()) {
-                    url = url.replace("&amp;", "&");
-                    datum = seite1.extract("<p class=\"dachzeile\">", "<", pos);
-                    datum = datum.replace("Uhr", "").trim();
-                    if (datum.contains("|")) {
-                        zeit = datum.substring(datum.indexOf('|') + 1).trim();
-                        zeit = zeit + ":00";
-                        datum = datum.substring(0, datum.indexOf('|')).trim();
-                    }
-                    titel = seite1.extract("<h4 class=\"headline\">", "<", pos);
-                    dauer = seite1.extract("<p class=\"subtitle\">", "<", pos);
-                    try {
-                        dauer = dauer.replace("Min.", "").trim();
-                        dauer = dauer.replace("| UT", "").trim();
-                        d = Long.parseLong(dauer) * 60;
-                    } catch (Exception ignored) {
-                    }
-                    if (d == 0) {
-                        Log.errorLog(915263621, "Dauer==0: " + strUrlFeed);
-                    }
-                    urlSendung = seite1.extract("<a href=\"/tv/", "\"", pos);
-                    if (!urlSendung.isEmpty()) {
-                        urlSendung = "http://www.ardmediathek.de/tv/" + urlSendung;
-                        urlSendung = urlSendung.replace("&amp;", "&");
-                    }
-
-                    filmSuchen2(url, thema, titel, d, datum, zeit, urlSendung);
+                if (url.equals(""))
+                {
+                    continue;
+                }
+                url = url.replace("&amp;", "&");
+                datum = seite1.extract("<p class=\"dachzeile\">", "<", pos);
+                datum = datum.replace("Uhr", "").trim();
+                if (datum.contains("|"))
+                {
+                    zeit = datum.substring(datum.indexOf("|") + 1).trim();
+                    zeit = zeit + ":00";
+                    datum = datum.substring(0, datum.indexOf("|")).trim();
+                }
+                titel = seite1.extract("<h4 class=\"headline\">", "<", pos);
+                dauer = seite1.extract("<p class=\"subtitle\">", "<", pos);
+                try
+                {
+                    dauer = dauer.replace("Min.", "").trim();
+                    dauer = dauer.replace("| UT", "").trim();
+                    d = Long.parseLong(dauer) * 60;
+                } catch (Exception ex)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1166)